lib/tasks/cleanup.rake

```diff
@@ -68,6 +68,7 @@ namespace :cleanup do
 
     puts 'Authorship rebuilt in all publications associated with the primary profile'
 
+    primary_author.update(orcidid: duped_author.orcidid)
     duped_author.cap_import_enabled = false
```

Comment:
rake task...when merging authors, be sure the ORCID is set on the primary author row too